ivy-scaling-nginx/README.md

```diff
@@ -0,0 +1,22 @@
+# ivy-scaling-nginx
+
+This example shows how to scale Axon Ivy Engine with docker-compose and NGINX.
+
+![Scaling NGINX](scaling-nginx.png)
+
+After starting this example you can simply go to http://localhost
+
+This example pre-configures an Ivy cluster with three nodes and configures them with sticky sessions (via ip_hash).
```

Comment:
I can only see one node in the docker-compose

Reply:
Changed README